rendered/zip-0230.html

```diff
@@ -433,6 +386,73 @@
+                            <th>Description</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td><code>varies</code></td>
+                            <td><code>nActionsOrchard</code></td>
+                            <td><code>compactSize</code></td>
+                            <td>The number of Action descriptions in <code>vActionsOrchard</code>.</td>
+                        </tr>
+                        <tr>
+                            <td><code>852 * nActionsOrchard</code></td>
+                            <td><code>vActionsOrchard</code></td>
+                            <td><code>OrchardZsaAction[nActionsOrchard]</code></td>
+                            <td>A sequence of ZSA Swap Action descriptions in the Action Group.</td>
+                        </tr>
+                        <tr>
+                            <td><code>1</code></td>
+                            <td><code>flagsOrchard</code></td>
+                            <td><code>byte</code></td>
+                            <td>As defined in Section 7.1 of the Protocol Specification <a id="footnote-reference-17" class="footnote_reference" href="#protocol-txnencoding">6</a>.</td>
+                        </tr>
+                        <tr>
+                            <td><code>32</code></td>
+                            <td><code>anchorOrchard</code></td>
+                            <td><code>byte[32]</code></td>
+                            <td>As defined in Section 7.1 of the Protocol Specification <a id="footnote-reference-18" class="footnote_reference" href="#protocol-txnencoding">6</a>.</td>
+                        </tr>
+                        <tr>
+                            <td><code>varies</code></td>
+                            <td><code>sizeProofsOrchard</code></td>
+                            <td><code>compactSize</code></td>
+                            <td>As defined in Section 7.1 of the Protocol Specification <a id="footnote-reference-19" class="footnote_reference" href="#protocol-txnencoding">6</a>.</td>
+                        </tr>
+                        <tr>
+                            <td><code>sizeProofsOrchard</code></td>
+                            <td><code>proofsOrchard</code></td>
+                            <td><code>byte[sizeProofsOrchard]</code></td>
+                            <td>The aggregated zk-SNARK proof for all Actions in the Action Group.</td>
+                        </tr>
+                        <tr>
+                            <td><code>4</code></td>
+                            <td><code>timeLimit</code></td>
+                            <td><code>uint32</code></td>
+                            <td>The block number (in the future) after which the Actions of the Action Group become invalid and should be rejected by consensus. This is set to be a value of <code>0</code>.</td>
```

Comment:
"This is set to be a value of <code>0</code>." I do not understand what you would like to say.